official/transformer/v2/transformer_benchmark.py

```diff
@@ -150,6 +151,54 @@ def benchmark_1_gpu(self):
                                    bleu_min=25.3,
                                    bleu_max=26)
 
+  def benchmark_8_gpu(self):
+    """Benchmark 8 gpu.
+
+      The paper uses 8 GPUs and a much larger effective batch size, this is will
+      not converge to the 27.3 BLEU (uncased) SOTA.
```

Comment:
Change this to say:
"Should converge to 27.3 BLEU (uncased). This has not been confirmed yet."